jams/nsconvert.py

```diff
@@ -167,7 +168,9 @@ def note_hz_to_midi(annotation):
     '''Convert a pitch_hz annotation to pitch_midi'''
 
     annotation.namespace = 'note_midi'
-    annotation.data.value = 12 * (np.log2(annotation.data.value) - np.log2(440.0)) + 69
+
+    logf = np.log2(np.asarray(annotation.data.value.values, dtype=np.float))
```

Comment:
nit: any value to breaking this pitch-hz mapping out as a separate function (in the spirit of do one thing)